src/main/java/com/Skynet/teacher/entities/Aula.java

```diff
@@ -25,13 +25,12 @@
 	
 	@ManyToOne
 	@JoinColumn(name = "disciplina_turma_id")
-	@JsonBackReference
+	@JsonIgnore
```

Comment:
Pode me explicar para qus erve o Jsonignore, pq nao percisamos da disciplinaturma ?

Reply:
https://stackoverflow.com/questions/17393812/json-and-java-circular-reference

Estavamos com um problema, que é um bug monstro da biblioteca do Jackson, que faz com que o @JsonManagedReference E @JsonBackReference, o Json ignore é uma segunda opção pra corrigir esse bug.